wolfssl/wolfcrypt/asn.h

```diff
@@ -2737,6 +2748,12 @@ struct OcspResponse {
     byte*   response;        /* Pointer to beginning of OCSP Response */
     word32  responseSz;      /* length of the OCSP Response */
 
+    enum responderIdType responderIdType;
```

Comment:
Does this new union need to always be part of the OcspResponse structure or can it be just with compatibility enabled? I am worried about struct size growth on stack.

Reply:
It's always needed. If stack size is a concern I can refactor the code to use heap memory instead.